Buildfile

```diff
@@ -8,5 +8,5 @@ cookbook.depends 'turnstile' do |turnstile|
 end
 
 profile :default do |default|
-  default.chef.run_list 'turnstile::default'
+  default.chef.run_list ['turnstile::nodejs', 'turnstile::default']
```

Comment:
The reason I included the `turnstile::nodejs` recipe in the `Buildfile` run_list instead of directly included in the default recipe is exactly like you said: because it gives the option to install Node in our Turnstile-approved way but doesn't require it.

This should be a major bump, for sure.